CHANGELOG.md

```diff
@@ -206,6 +206,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* [\#11630](https://github.com/cosmos/cosmos-sdk/pull/11630) Added SafeSub method to avoid panic in Sub method for sdk.Coin
```

Comment:
I'm not sure this is a bug fix per se. I'd probably move this to `Improvements` IMO